src/Orleans.Transactions/InClusterTM/TransactionManager.cs

```diff
@@ -5,10 +5,99 @@
 using System.Threading;
 using Microsoft.Extensions.Options;
 using Microsoft.Extensions.Logging;
+using Orleans.Runtime;
 using Orleans.Transactions.Abstractions;
+using Orleans.Runtime.Configuration;
 
 namespace Orleans.Transactions
 {
+    internal class TransactionManagerMetrics
+    {
+        private const string StartTransactionRequestTPS = "TransactionManager.StartTransaction.TPS";
```

Comment:
I guess you followed what was done in `TransactionAgentMetrics`, could these string values be useful for the developer?

Reply:
Agent is one per silo, this TransactionManager is a grain. Agent send batch request to TM to start transactions, and commit transactions. So even the metric name maybe similar as the one in AgentMetrics, I think they are measuring different metrics. It isn't based on AgentMetrics. 

I think it will be useful to developer. For example, when we see transaction are not being processed fast enough, we can look at TM metrics to see what's the TPS handling here, if at peak, maybe the bottle neck is TM. Otherwise, the bottle neck is else where.

Reply:
So it they are useful for the dev, should we put these as public somewhere?

Reply:
Ah I see, this is similar issue with all of our internal metrics, there's no easy way to figure out what metric string are reporting.  We can have a Metrics.cs file listing all the metrics in each assembly.